Move lineage AWS IAM permissions into compute env and manual setup pages (#1401)
seqeralabs/docs: platform-cloud/docs/compute-envs/aws-batch.md

}
```

### Data lineage (optional)

> If [data lineage](../data/data-lineage) is enabled in your workspace, the Platform integration credentials require the following additional permissions to create the queue infrastructure and bucket notifications used by the lineage service:

Reviewer: ```suggestion
If you enable [data lineage](../data/data-lineage) in your workspace, add the following permissions to your Platform integration credentials to create the queue infrastructure and bucket notifications used by the lineage service:
```

Author: @robnewman could you confirm whether data lineage will be an opt-in feature?

Reviewer 2: @bebosudo, by opt-in do you mean not on by default?

Author: Exactly, the beginning of the sentence ("if you enable data lineage") makes me think it's a feature that needs to be enabled somehow?

If it's already enabled when Platform v26.1 is used, shall we rename it to "If you use data lineage ..." ?

```json
{
  "Sid": "LineageIntegrationSQS",
  "Effect": "Allow",
  "Action": [
    "sqs:CreateQueue",
    "sqs:GetQueueAttributes",
    "sqs:SetQueueAttributes",
    "sqs:GetQueueUrl",
    "sqs:ReceiveMessage",
    "sqs:DeleteMessage"
  ],
  "Resource": "arn:aws:sqs:<REGION>:<ACCOUNT_ID>:seqera-lineage-*"
},
{
  "Sid": "LineageIntegrationS3",
  "Effect": "Allow",
  "Action": [
    "s3:CreateBucket",
    "s3:GetBucketNotificationConfiguration",
    "s3:PutBucketNotificationConfiguration",
    "s3:GetBucketLocation"
  ],
  "Resource": "arn:aws:s3:::seqera-lineage-*"
}
```

> If you manage your own EC2 instance role or head job role (rather than letting Seqera create them with Batch Forge), see [Manual AWS Batch configuration](../enterprise/advanced-topics/manual-aws-batch-setup#create-an-ec2-instance-role) for the additional S3 permissions required on those roles.

Reviewer: ```suggestion
If you manage your own EC2 instance role or head job role (rather than letting Seqera create them with Batch Forge), see [Manual AWS Batch configuration](../enterprise/advanced-topics/manual-aws-batch-setup#create-an-ec2-instance-role) for additional S3 permissions to add to those roles.
```
